loklak.php

```diff
@@ -186,4 +186,54 @@ public function suggest($query="", $count=null, $order=null, $orderby=null, $sin
 			return json_encode($request, true);
 		}
 	}
+	
+	public function aggregations($query="", $since=null, $until=null, $fields=null, $limit=null) {
+		$this->requestURL = $this->baseUrl . '/api/search.json';
+		$this->query = $query;
+		$this->since = $since;
+		$this->until = $until;
+		$this->fields = $fields;
+		$this->$limit = $limit;
+		if ($query) {
+			$params = array('q'=>$this->query);
+			if ($since) {
+				$params['q'] = $params['q'] . ' since:'.$this->since;
+			}
+			if ($until) {
+				$params['q'] = $params['q'] . ' until:'.$this->until;
+			}
+			if ($fields) {
+				$field_string = "";
+				foreach ($this->fields as $field) {
+					$field_string += $field . ',';
```

Comment:
Given the `array()` of fields as `("mentions","hashtags")` wouldn't this line construct it as `mentions,hashtags,` notice the extra `,` at the end, This doesn't cause an error but probably isn't what we really want to have in the string right ?